src/Components/DnDLayout/GridTile.tsx

```diff
@@ -162,7 +162,7 @@ const GridTile = ({ widgetType, title, isDragging, setIsDragging, setWidgetAttri
         </CardTitle>
       </CardHeader>
       <Divider />
-      <CardBody>
+      <CardBody className="pf-v5-u-p-0">
```

Comment:
@florkbr @epwinchell do we want the card body padding to be there by default and handle by each widget separately? I removed it for the table to look properly, but we can decide for a different approach

Reply:
@fhlavac I removed it on master